### PR TITLE
feat/main_4

### PR DESCRIPTION
**Pull request by Quannichan**
feat: new design of main's component 4

### CODE REVIEW

File: src/components/main_compo_4.html [hunk 1]
```diff
@@ -1 +1,96 @@
...
+            document.getElementById("plan3").style.transform= "translateY(30px)";
+            document.getElementById(ID_BTN).style.backgroundColor="#EBCB80";
+            document.getElementById(ID_BTN).style.color="#4B4B4B";
+            document.getElementById("btn_pl1").style.backgroundColor ="transparent";
+            document.getElementById("btn_pl1").style.color ="black";
+            document.getElementById("plan1").style.padding = "5% 8%"
+            document.getElementById("plan3").style.padding = "5% 5% "
+            document.getElementById("plan2").style.padding = "5% 6% "
+            document.getElementById("btn_pl2").style.backgroundColor ="transparent";
+            document.getElementById("btn_pl2").style.color ="black";
```
Comment (Lanng21) [inline src/components/main_compo_4.html:93]: why u use JS here? We are practicing html css, and it is not necessary to use js.

File: src/layouts/main.html [hunk 2]
```diff
@@ -1 +1,14 @@
...
+    <!-- include main component 1 -->
+    <include src="./components/main_compo_1.html"></include>
+    <!-- include main component 2 -->
+    <include src="./components/main_compo_2.html"></include>
+    <!-- include main component 3 -->
+    <include src="./components/main_compo_3.html"></include>
+    <!-- include main component 4 -->
+    <include src="./components/main_compo_4.html"></include>
+</main>
+>>>>>>> Stashed changes
```
Comment (Lanng21) [inline src/layouts/main.html]: pls double check

File: src/stylesheet/_mixins.scss [hunk 3]
```diff
@@ -1 +1,465 @@
...
+@mixin btn_pl_phone{
+    padding: 5% 15%;
+    font-size: 11px;
+    font-weight: 400;
+    line-height: normal;
+    letter-spacing: 0.8px;
+    border-style: none;
+    border-radius: 20px;
+}
+>>>>>>> Stashed changes
```
Comment (Lanng21) [inline src/stylesheet/_mixins.scss]: same

File: src/layouts/main.html (hunk 2 above)
Comment (Quannichan) [reply src/layouts/main.html]: I 've solved it

File: src/components/main_compo_4.html (hunk 1 above)
Comment (Quannichan) [reply src/components/main_compo_4.html:93]: I use js for onclick animation.

**Review by Lanng21 — COMMENTED** [review]
naming branch, PR, file, class verry bad with me. main 1, 2, 3? u analysis design and u defined naming for section are main1 main2 main3?

File: src/components/main_compo_4.html (hunk 1 above)
Comment (Lanng21) [reply src/components/main_compo_4.html:93]: no, it's not necessary. you should do everything in html css.

**Pull request closed without merging**